std_msgs/Header header
uint32 timestamp
uint32 frame_counter
float32 shutter
float32 gain
uint16  wb_blue
uint16  wb_red
